Portfolio2.css

```diff
@@ -0,0 +1,222 @@
+* {
+	margin:0;
+	padding: 0;
+	
+}
+
+body {
+	background-image: url("https://wallpaperaccess.com/full/1128684.jpg");
+	background-size: cover;
+	
+	
+ }
+
+header {
+	display: inline-flex; 
+	position: fixed; 
+	margin-left: 1200px;
+	position:fixed;
+	margin-top:-180px;
```

Comment:
Take out these margins.

When you use position: fixed you can give exact co-ordinates to position the element. So instead of giving negative top margin (if you ever need to use negative margin, it's a sign that it could be setup in a better way) and loads of left margin, you can just set these rules:

top: 0;
right: 0;

And that will position it at the top right corner of the page at all times.